src/main/java/dao/PersonDao.java

```diff
@@ -35,4 +37,10 @@ public void deleteById(Long id) {
     }
 
     public void update(Person person){currentSession().update(person);}
+
+    public Person getUserByEmail(String email) {
```

Comment:
For methods which return a single value and can possibly not find the result, it's a good practice to return an [Optional](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html) value.